storage/raft.go

```diff
@@ -48,68 +48,59 @@ type raftLogger struct {
 	group uint64
 }
 
-func (r *raftLogger) prependContext(format string, v []interface{}) string {
-	var s string
+func (r *raftLogger) context() string {
 	if r.group != 0 {
-		v2 := append([]interface{}{r.group}, v...)
-		s = fmt.Sprintf("[group %d] "+format, v2...)
-	} else {
-		s = fmt.Sprintf(format, v...)
+		return fmt.Sprintf("[group %d] ", r.group)
 	}
-	return s
+	return ""
 }
 
 func (*raftLogger) Debug(v ...interface{}) {
 	if log.V(2) {
-		log.InfoDepth(1, v...)
+		log.InfoDepth(1, "", v...)
 	}
 }
 
 func (r *raftLogger) Debugf(format string, v ...interface{}) {
 	if log.V(2) {
-		s := r.prependContext(format, v)
-		log.InfoDepth(1, s)
+		log.InfoDepth(1, r.context()+format, v...)
 	}
 }
 
 func (*raftLogger) Info(v ...interface{}) {
 	if log.V(1) {
-		log.InfoDepth(1, v...)
+		log.InfoDepth(1, "", v...)
```

Comment:
Unrelated to this change, but the non-`f` logging functions should really prepend `context()` to their arguments. (I'm not sure if these are actually used). Maybe leave a TODO.